lib/mayaUsd/utils/layers.cpp

```diff
@@ -116,16 +116,33 @@ void enforceMutedLayer(const PXR_NS::UsdPrim& prim, const char* command)
     }
 }
 
+static SdfPrimSpecHandleVector _GetLocalPrimStack(const UsdPrim& prim)
```

Comment:
ON second thought, maybe adding a comments on what this code is trying to avoid, like the description you gave in teh bug would be helpful for future coders.

Reply:
I too the liberty of adding the comment myself to your branch.

Reply:
Thank you!!